contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -230,58 +327,90 @@ NXmpes(NXobject):
             This is the momentum axis to be used for data plotting.
     (NXsample):
       name:
-      chemical_formula:
-        exists: recommended
-        doc: |
-          The chemical formula of the sample. For mixtures use the NXsample_component
-          group in NXsample instead.
-      sample_history(NXnote):
-        exists: recommended
+      (NXsubstance):
         doc: |
-          A descriptor to keep track of the treatment of the sample before entering the
-          photoemission experiment. Ideally, a full report of the previous operations, in
-          any format (NXnote allows to add pictures, audio, movies). Alternatively, a
-          reference to the location or a unique identifier or other metadata file. In the
-          case these are not available, free-text description.
+          For samples containing a single pure substances. For mixtures use the
+          NXsample_component_set and NXsample_component group in NXsample instead.
```

Comment:
NXsubstance is required. How does this work with NXsample_component_set then? Not clear how to use it, and what fields to provide.

Reply:
You are right, I thought in the existing NXmpes, `chemical_formula` was required. Since it is only recommended, we can also recommend NXsubstance.